db/migrations/20221205161456_users.sql

```diff
@@ -0,0 +1,20 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT,
+		last_name VARCHAR(50) NOT NULL,
+    first_name VARCHAR(50) NOT NULL,
+    date_of_birth VARCHAR(50) NOT NULL,
+    phone_number VARCHAR(50) NOT NULL,
+    gender VARCHAR(50) NOT NULL,
+    address VARCHAR(50) NOT NULL,
+		email VARCHAR(50) NOT NULL,
+		password VARCHAR(100) NOT NULL,
+    point DECIMAL,
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+		CONSTRAINT users_email_ukey UNIQUE (email),
+		PRIMARY KEY(id)
+);
```

Comment:
전반적인 들여쓰기 (Indentation)를 수정하여 코드의 가독성을 높여주세요~!
- id 컬럼에 PRIMARY KEY 속성을 지정하는 경우 아래 코드와 같이 작성할 수도 있습니다. 참고해주세요~!
- date_of_birth 컬럼은 birhdate 로 컬럼명을 수정해도 좋을 것 같습니다!
- email 컬럼의 최대 길이를 수정하는 것이 좋을 것 같습니다! 이메일의 최대 길이에 대한 자료를 찾아보고 적용해주세요!
- bcrypt를 활용하여 비밀번호를 암호화하는 경우 반환되는 데이터 타입과 최대 길이를 찾아보고 적용해주세요!

```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    last_name VARCHAR(50) NOT NULL,
    first_name VARCHAR(50) NOT NULL,
    birthdate VARCHAR(50) NOT NULL,
    phone_number VARCHAR(50) NOT NULL,
    gender VARCHAR(50) NOT NULL,
    address VARCHAR(50) NOT NULL,
    email VARCHAR(255) NOT NULL,
    password BINARY(60) NOT NULL,
    point DECIMAL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE (email),
);
```